{{ cookiecutter.project_slug }}/{{ cookiecutter.project_module }}/urls.py

```diff
@@ -17,11 +17,13 @@
 from django.urls import path, include
 
 import automationcommon.views
+from django.views.generic import TemplateView
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', include('ucamwebauth.urls')),
     path('status', automationcommon.views.status, name='status'),
+    path('healthz', TemplateView.as_view(template_name="healthz.html"), name='healthz'),
```

Comment:
See elsewhere in this commit. This could be as simple as:
```python
from django.http import HttpResponse

# ...
path('healthz', lambda request: HttpResponse('ok', content_type="text/plain"), name='healthz'),
```

although using a ``lambda`` is a bit icky.